src/main/java/wutdahack/actuallyunbreaking/mixin/UnbreakingEnchantmentMixin.java

```diff
@@ -0,0 +1,50 @@
+package wutdahack.actuallyunbreaking.mixin;
+
+import amymialee.noenchantcap.NoEnchantCap;
+import java.util.Random;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.enchantment.EnchantmentTarget;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.enchantment.MendingEnchantment;
+import net.minecraft.enchantment.UnbreakingEnchantment;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+import wutdahack.actuallyunbreaking.AUConfig;
+
+@Mixin(UnbreakingEnchantment.class)
+abstract class UnbreakingEnchantmentMixin extends Enchantment {
+
+    // items can't have mending and unbreaking together
```

Comment:
Why is that? One might wish to enchant an unbreakable tool with lost durability with Mending in order to remove the damage bar.

Reply:
In the ItemStackMixin, I used the setDamage method in order to remove the damage bar, so there was no point to use Mending with Unbreaking.

Reply:
I see. I will fix that.